jormungandr/src/rest/v1/logic.rs

```diff
@@ -111,3 +114,49 @@ pub async fn get_fragment_logs(context: &Context) -> Result<Vec<FragmentLog>, Er
     .instrument(span)
     .await
 }
+
+pub async fn get_account_votes(
+    context: &Context,
+    vote_plan_id: VotePlanId,
+    address: Address,
+) -> Result<Option<Vec<u8>>, Error> {
+    let span = span!(parent: context.span()?, Level::TRACE, "get_account_votes", request = "get_account_votes");
+
+    let address: chain_addr::Address = address.into();
+    let identifier = match address.kind() {
+        chain_addr::Kind::Account(pubkey) => {
+            let account_id = chain_impl_mockchain::account::Identifier::from(pubkey.clone());
+            chain_impl_mockchain::transaction::UnspecifiedAccountIdentifier::from_single_account(
+                account_id,
+            )
+        }
+        _ => return Err(Error::UnexpectAddressType),
+    };
+
+    let vote_plan_id: chain_crypto::digest::DigestOf<_, _> = vote_plan_id.into_digest().into();
+
+    async move {
+        let maybe_vote_plan = context
+            .blockchain_tip()?
+            .get_ref()
+            .await
+            .ledger()
+            .active_vote_plans()
+            .into_iter()
+            .find(|x| x.id == vote_plan_id);
+        let vote_plan = match maybe_vote_plan {
+            Some(vote_plan) => vote_plan,
+            None => return Ok(None),
```

Comment:
Should we respond with 404 when the vote plan is not found?

Reply:
It is converted into 404 in `handlers.rs` the same way we do in `v0` since statuses handling is specific to a particular framework.

Reply:
OK. It could be another internal error case instead.